iaso/management/commands/fix_submissions_select_multiple.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from datetime import datetime
+import iaso.models as m
+
+# This is one time command to "reparse" the xml to json
+# All submissions for forms with select_multiple has to be reparse
+# the select_multiple questions have been ignored since the merge/deploy of https://github.com/BLSQ/iaso/pull/1265
+#
+# testing on a small subset on your dev env
+#
+#    docker compose run --rm iaso manage seed_test_data --mode=seed --dhis2version=2.40.4.1
+#    docker-compose run --rm iaso manage fix_submissions_select_multiple
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        date_limite = datetime(2024, 6, 1)
+
+        for form in m.Form.objects.all():
```

Comment:
Is the `Form` table a big one?

Because if it's big [using `iterator` with a `chunk_size`](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#iterator) might be a better solution?